lib/rubyapi_rdoc_generator.rb

```diff
@@ -54,6 +52,10 @@ def generate_objects
             depth: constant_depth(doc.full_name)
           }
         }
+
+        next if methods.find { |m| m[:name] == method[:name] && m[:method_type] == method[:method_type] }
```

Comment:
```suggestion
        next if methods.any? { |m| m[:name] == method[:name] && m[:method_type] == method[:method_type] }
```

Since you don't need the return value, it's quite a bit faster to use `any?` (which just returns a boolean), than `find` (which returns the found element).

```ruby
require 'benchmark/ips'

input = 5
vals = 1.upto(10).map { |val| { v: val } }

Benchmark.ips do |x|
  x.report('using #find') do
    vals.find { |val| val[:v] == input }
  end

  x.report('using #any?') do
    vals.any? { |val| val[:v] == input }
  end

  x.compare!
end
```

```
Warming up --------------------------------------
         using #find   191.311k i/100ms
         using #any?   334.715k i/100ms
Calculating -------------------------------------
         using #find      1.927M (± 6.5%) i/s -      9.757M in   5.089815s
         using #any?      3.305M (± 4.3%) i/s -     16.736M in   5.075868s

Comparison:
         using #any?:  3304660.9 i/s
         using #find:  1927283.9 i/s - 1.71x  (± 0.00) slower
```

Reply:
Ah thanks!